pandas/tests/io/formats/test_to_csv.py

```diff
@@ -541,23 +543,29 @@ def test_to_csv_compression_dict_no_method_raises(self):
                 df.to_csv(path, compression=compression)
 
     @pytest.mark.parametrize("compression", ["zip", "infer"])
-    @pytest.mark.parametrize(
-        "archive_name", [None, "test_to_csv.csv", "test_to_csv.zip"]
-    )
+    @pytest.mark.parametrize("archive_name", ["test_to_csv.csv", "test_to_csv.zip"])
```

Comment:
what happened to the `None` case?

Reply:
That's now being tested in the newly-added test case below.
Reason being that coming up with the expected inner file name now got a tiny bit trickier, and testing this all in this one test would get too convoluted.